dev/source/docs/mavlink-get-set-home-and-origin.rst

```diff
@@ -41,81 +39,160 @@ The message definition can be found `here <https://mavlink.io/en/messages/common
    <td>System ID</td>
    </tr>
    <tr>
-   <td><strong>latitude</strong></td>
-   <td>int32_t</td>
-   <td>Latitude * 1e7</td>
+   <td><strong>target_component</strong></td>
```

Comment:
This is pre-existing, but this really ought to be using an rst table rather than html.  I'd stand a chance of understanding this if it was, too :-)

Reply:
yeah, sorry, pre-existing so I'll leave it as it is if that's ok.